src/nanoplot/config.vsh.yaml

```diff
@@ -0,0 +1,64 @@
+name: nanoplot
+description: |
+  Run NanoPlot on nanopore-sequenced reads.
+  "Sample"
+keywords: ["quality control", "qc", "fastq", "sequencing summary", "nanopore"]
+links:
+  repository: https://github.com/wdecoster/NanoPlot/tree/master
+  homepage: http://nanoplot.bioinf.be/
+license: MIT
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --fastq
+        type: file
+        description: List of input basecalled-FastQ files.
+        example: read.fq
+        multiple: true
+      - name: --summary
+        type: file
+        description: Summary files.
+        example: read.txt
+        multiple: true
+      - name: --bam
+        type: file
+        description: BAM files.
+        example: read.bam
+        multiple: true
+  - name: Outputs
+    arguments:
+      - name: --txt
+        type: file
+        description: Stats from NanoPlot
+        direction: output
+        example: NanoStats.txt
+      - name: --png
+        type: file
+        description: Plots generated by NanoPlot
+        direction: output
+        example: plots.png
+      - name: --html
+        type: file
+        description: NanoPlot report
+        direction: output
+        example: report.html
+resources:
+  - type: bash_script
+    path: script.sh
+test_resources:
+  - type: bash_script
+    path: test.sh
+  - type: file
+    path: test_data
+engines:
+  - type: docker
+    image: quay.io/biocontainers/nanoplot:1.42.0--pyhdfd78af_0
+    setup:
+      - type: docker
+        run: |
+          touch ./var/software_versions.txt && \
+          version=$(docker run quay.io/biocontainers/nanoplot:1.42.0--pyhdfd78af_0 NanoPlot --version) && \
+          echo "$version" > ./var/software_versions.txt
```

Comment:
* This code is run inside the docker. You're running docker in docker with this command.
* Make sure that the format of the resulting file is valid yaml, i.e. the contents of the file should be `NanoPlot: "1.42.0"`.
* Output the file to `/var/software_versions.txt`, not `/var/software_versions.txt`

Check the other components for valid examples

Reply:
I think I have output the file to the correct destination?